docs/development/visualize/development-create-visualization.asciidoc

```diff
@@ -466,4 +466,4 @@ the field. The type can be either 'text' or 'html'.
 - *getValue(bucket)* : gets value for a specific bucket
 - *getField()* : gets the field used for this aggregation
 - *getFieldDisplayName()* : gets field display name
-- *getAggParams()* : gets the arguments to the aggregation
\ No newline at end of file
+- *getAggParams()* : gets the arguments to the aggregation
```

Comment:
This shows up weirdly in the diff, but it's adding a newline at the end of the file.